src/services/ofac.service.ts

```diff
@@ -0,0 +1,31 @@
+import dotenv from "dotenv"
+
+dotenv.config()
```

Comment:
I would like this to be as a service class that accepts url and api key arguments and doesn't deal with env.
The env should be processed in the app startup file.